assets/stylesheets/shared/_color-schemes.scss

```diff
@@ -26,5 +36,10 @@
 		--masterbar-item-hover-background: darken( $gray, 10% );
 		--masterbar-item-active-background: $gray-text-min;
 		--masterbar-item-new-color: $gray-dark;
+		--masterbar-item-new-editor-background: $gray-text-min;
+		--masterbar-item-new-editor-hover-background: lighten( $gray-text-min, 5% );
+		--masterbar-toggle-drafts-editor-background: darken( $gray, 10% );
+		--masterbar-toggle-drafts-editor-border: $gray-dark;
+		--masterbar-toggle-drafts-editor-hover-background: lighten( $gray-text-min, 5% );
```

Comment:
There's a bit of a semantics issue here, though it's outside of the scope of this PR...
We're perpetuating the misuse of the `$gray-text-min` variable (in terms of semantics).
Merely an observation here - FFTI :)

Reply:
I'd agree 👍 I'll swap that out in the context of the Sidebar PR as I'm reworking the `_color-schemes.scss` quite a bit there.